### DISCUSSION TITLE
Key (object_type_id)=(55) is not present in table "django_content_type"

### DISCUSSION THREAD

**Discussion by jepify** [netbox-community/netbox#14194] [Help Wanted!]
### NetBox version

v3.6.4

### Python version

3.11

### Steps to Reproduce

Can not reproduce yet in a simple case. We are getting this issue during our testing in our pipelines.

We are simply creating a device with the pynetbox api
``` 
device = nb.dcim.devices.create(
                name=name,
                device_type=d_type.id,
                role=role.id,
                site=site.id,
                asset_tag=asset_tag,
                tags=tag_ids,
                serial=f"{serial_base}{number}",
                tenant=test_tenant.id,
                rack=rack,
            )
```

### Expected Behavior

The device to be created

### Observed Behavior

 pynetbox.core.query.RequestError: The request failed with code 500 Internal Server Error: {'error': 'insert or update on table "extras_cachedvalue" violates foreign key constraint "extras_cachedvalue_object_type_id_6f47d444_fk_django_co"\nDETAIL:  Key (object_type_id)=(55) is not present in table "django_content_type".', 'exception': 'IntegrityError', 'netbox_version': '3.6.4', 'python_version': '3.11.4'}

**Comment by gustavi** [comment]
@jepify did you find any solution to this problem?

**Comment by candlerb** [comment]
A starting point would be to get a full backtrace: either set ADMINS and EMAIL in configuration.py (in which case it will get emailed to you), or set DEBUG=True temporarily (in which case you get it via the web interface).

The question is, why is Netbox trying to insert or update a row on "extras_cachedvalue" with content type 55, if no such content type exists in the "django_content_type" table? Where does it get 55 from? The backtrace may help explain.

Do you have any plugins installed, or have you ever had any plugins installed? In particular, have you used any secrets plugin?

You could also try rebuilding the search cache:

```
/opt/netbox/venv/bin/python /opt/netbox/netbox/manage.py reindex
```

Also check your django_content_type table for any obvious errors:

```
echo 'select * from django_content_type' | sudo -u postgres psql netbox
```

For me, id 55 is `secrets.secret`, but this can vary between installations.

I have 140 in total, including 8 from plugins:

```
echo 'select app_label,count(*) from django_content_type group by app_label order by app_label' | sudo -u postgres psql netbox

       app_label       | count
-----------------------+-------
 account               |     1
 admin                 |     1
 auth                  |     3
 circuits              |     6
 contenttypes          |     1
 core                  |     5
 dcim                  |    42
 django_rq             |     1
 extras                |    23
 ipam                  |    18
 netbox_animal_sounds  |     1
 netbox_topology_views |     7
 secrets               |     4
 sessions              |     1
 social_django         |     5
 taggit                |     2
 tenancy               |     6
 users                 |     5
 virtualization        |     5
 wireless              |     3
```

Some of these are legacy, e.g. "secrets" tables are hangovers from an old version of Netbox (and I'm not using any secrets plugin)
